### DISCUSSION TITLE
change account on GEE plugin

### DISCUSSION THREAD

**Discussion by Newton787** [gee-community/qgis-earthengine-plugin#138] [General]
I use Gee plugin with my personal e-mail, but now i need use the professional email to access assets of my work, how i change the account?

**Comment by XavierCLL** [comment]
Hi @Newton787 

To reset your Google Earth Engine credentials: Delete the credentials file and re-authenticate. The credentials file is located at:

> Windows: C:\Users\USER\.config\earthengine\credentials 
> Linux: /home/USER/.config/earthengine/credentials 
> MacOS: /Users/USER/.config/earthengine/credentials

**Comment by gena** [comment]
A recommended way for this is to install the Earth Engine CLI and run `earthengine authenticate` (https://developers.google.com/earth-engine/guides/command_line#authenticate) and eventually `earthengine set_project` (https://developers.google.com/earth-engine/guides/command_line#set_project). Also, it does not have to be installed in the QGIS Python environment.
